### DISCUSSION TITLE
Some issues with MQTT

### DISCUSSION THREAD

**Discussion by ektoekto** [tillsteinbach/WeConnect-mqtt#350] [General]
First of all I want to thank you for the program, it did exactly what I was looking for. The only issue I'm having is that all messages containing strings are not valid JSON-objects, at least according to my Mosquito Nodered Broker. Not a big issue but there it is. I have about 40 different MQTT devices and all of them are capable of pushing messages as JSON-objects. 

Also the service install instructions do not work with the latest version, the example string has unknown parameter. I think it's this
" -m weconnect_mqtt ". And installation of Python has changed, it's done differently nowadays with virtualization and so on, and then all the locations are different. This is just so that you might remove instructions and point to a up-to-date source.

**Comment by pahenning** [comment]
First of all you should also write what type of car you have.
Secondly, there are numerous ways to install and update Python, and although virtualization is a good idea it is not absolutely necessary.  Python installation should be kept out of this documentation.
Well, and concerning the MQTT side: I cannot confirm this, I do not receive any complaint by my MQTT server (which is a FHEM MQTT2_Server).

pah
